src/ethereum_test_tools/common/types.py

```diff
@@ -845,30 +845,16 @@ class FixtureWithdrawal(Withdrawal):
     the beacon chain in the output fixture.
     """
 
-    index: NumberConvertible = field(
-        json_encoder=JSONEncoder.Field(
-            cast_type=ZeroPaddedHexNumber,
-        ),
-    )
-    validator: NumberConvertible = field(
-        json_encoder=JSONEncoder.Field(
-            name="validatorIndex",
-            cast_type=ZeroPaddedHexNumber,
-        ),
-    )
-    amount: NumberConvertible = field(
-        json_encoder=JSONEncoder.Field(
-            cast_type=ZeroPaddedHexNumber,
-        ),
-    )
```

Comment:
The `engineNewPayload.executionPayload.withdrawals` requires that these three fields are not zero padded.

Now this means the fixture generation does not zero pad these fields. By removing these, we simply inherit these fields from `Withdrawals` as they are not zero padded.

I think I should fix this before we merge.

Reply:
Fixture generation is fine, I'm just wondering if when these fields in the fixtures are not zero padded if they would break `evm blocktest` commands

Also it makes the fixtures looks slightly messy hehe

Going to tackle this first thing tmo